src/main/java/UseCaseClasses/ScheduleEditor.java

```diff
@@ -56,15 +56,19 @@ public void editEventDay(Event event, String newDay) {
      * @param user the Person whose schedule is being changed
      */
 
-    public void removeEvent(String eventName, String eventDay, Person user) {
+    public void removeEvent(String eventName, String eventDay, int startTime, Person user) {
         Schedule schedule = user.getUserSchedule();
 
-        HashMap<String, String> Day = schedule.getSchedule().get(eventDay);
-        for (int i = 0; i <= 23; ++i) {
-            if (Objects.equals(Day.get(String.valueOf(i)), eventName)){
-                Day.put(String.valueOf(i), null);
+        ArrayList<Event> dayevents = schedule.getSchedule().get(eventDay);
+        for (int i = 0; i < dayevents.size(); i++) {
+            if (Objects.equals(dayevents.get(i).eventName, eventName) && Objects.equals(dayevents.get(i).eventStartTime, startTime)) {

```

Comment:
we're not supposed to use objects.equals() -- this should be String.equals(), no?

Reply:
ah yes I can change that thanks

Reply:
Made a commit to change this